### PR TITLE
Buffs powersink because uhh idk apparently it keeps going kaboom or some shit due to the engines ooc requested it so here it is

### PR DESCRIPTION
**Pull request by Iamgoofball**
now can reach a max of ~~10e8~~1e42 power instead of 1e8 power before going kaboom

wtf does that even mean idk

pornhub told me i get 5 grand if I shill them in every PR

### CODE REVIEW

**Comment by PKPenguin321** [comment]
how much of an increase is that exactly

**Comment by Incoming5643** [comment]
10e8 = 1e9

Learn to science

**Comment by Iamgoofball** [comment]
literally have no clue

oh

ok ill buff it more

**Comment by Lumipharon** [comment]
goof, gotta increase the amount of power it can absort per tick or whever. because 4 smes' output more power then a powersink can suck down, or some shit.

**Comment by Iamgoofball** [comment]
ok

**Comment by Incoming5643** [comment]
https://en.wikipedia.org/wiki/Scientific_notation

**Comment by DaedalusGame** [comment]
1e8 = 100000000
1e24 = 1000000000000000000000000

????????

**Comment by Iamgoofball** [comment]
:joy:

**Comment by Lumipharon** [comment]
keep in mind with the tesla engine, you can safely have 23423409 balls, and produce rediculous amounts of energy, to the point where the powersink still can't do shit if the engine is wired to the grid.

**Comment by DaedalusGame** [comment]
okay so with your current setting it takes 202942668696 years for the powersink to explode

**Comment by Iamgoofball** [comment]
:joy: :joy: :joy:

**Comment by xxalpha** [comment]
Can you test this properly goof?

**Comment by PKPenguin321** [comment]
>:joy: :joy: :joy:
👌

**Comment by Iamgoofball** [comment]
:joy: :joy: :joy::joy: :joy: :joy:
      :joy:      :joy:

**Comment by Iamgoofball** [comment]
brb going to dutch bros

**Comment by CPTANT** [comment]
Anyway its a good change, powersinks should be found to disable them, not magically remove themselves all the time.

**Comment by Zilenan91** [comment]
Would it be possible to make powersink explosions depend on the amount of power they've leeched out and also make them have an ability to explode on command?

**Comment by MMMiracles** [comment]
@Zilenan91

>hook up powersink directly to singulo+solars for around 15-20 minutes
>create a bomb that blows a hole in half the station

singulo-powered bombs when

**Comment by Zilenan91** [comment]
That'd be fucking awesome. You can't pick powersinks up either so you could shoot the guy.

**Comment by Lumipharon** [comment]
You can pick up powersinks though.

Having no max limit to power it can abosrb would be good, since 99% of them get found before they blow anyway.

Being able to literally ignore the powersin because your engine makes so much power is gay though.

**Comment by neersighted** [comment]
I would consider hotwiring the engine to blow a powersink a feature, though normal SMES output should not do it and should drain power faster than it is emitted.

File: code/game/objects/items/devices/powersink.dm [hunk 1]
```diff
@@ -12,9 +12,9 @@
 	throw_range = 2
 	materials = list(MAT_METAL=750)
 	origin_tech = "powerstorage=3;syndicate=5"
-	var/drain_rate = 600000		// amount of power to drain per tick
+	var/drain_rate = 9000000		// amount of power to drain per tick
 	var/power_drained = 0 		// has drained this much power
-	var/max_power = 1e8		// maximum power that can be drained before exploding
+	var/max_power = 1e42		// maximum power that can be drained before exploding
```
Comment (neersighted) [inline code/game/objects/items/devices/powersink.dm]: Goofball please understand orders of magnitude.

**Comment by neersighted** [comment]
i ded

**Comment by duncathan** [comment]
Let's talk about how this makes powersinks hold 1,000,000,000,000,000,000,000,000,000,000,000 times as much power as before
That's one decillion times as much

that's a big number guys

**Comment by Jordie0608** [comment]
Fun fact, it would take 6000 solar panels at full sun exposure to meet this energy drain.
That'd be the combined solar power of 25 stations at once.
We don't have that many stations.

**Comment by Iamgoofball** [comment]
2 :heavy_multiplication_x: 2 = 5

**Comment by PKPenguin321** [comment]
this is my favorite PR ever, every time i look at it i crack up

**Pull request closed without merging**